dht-node/src/config/index.ts

```diff
@@ -0,0 +1,62 @@
+// ATTENTION: DO NOT PUT ANY SECRETS IN HERE (or the .env)
+import dotenv from 'dotenv'
+dotenv.config()
+
+const constants = {
+  DB_VERSION: '0059-add_hide_amount_to_users',
+  LOG4JS_CONFIG: 'log4js-config.json',
+  // default log level on production should be info
+  LOG_LEVEL: process.env.LOG_LEVEL || 'info',
+  CONFIG_VERSION: {
+    DEFAULT: 'DEFAULT',
+    EXPECTED: 'v1.2023-01-01',
+    CURRENT: '',
+  },
```

Comment:
Like we discused with @ulfgebhardt would be nice to refactor this into the database.